WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingViewModel.swift

```diff
@@ -19,15 +19,21 @@ final class StoreOnboardingViewModel: ObservableObject {
 
     @Published private(set) var taskViewModels: [TaskViewModel]
 
-    private let isExpanded: Bool
+    var numberOfTasksCompleted: Int {
```

Comment:
nit: might be nice unit testing this? I understand that we don't have unit tests for this class right now, feel free to leave it for the future

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/8948/commits/ef5b2ec11a259f11286faf3231dd6324a34a42f3 and https://github.com/woocommerce/woocommerce-ios/pull/8948/commits/dac2df6182e209dbc1aee5b2df0a29416baa7c43